.changeset/late-mice-type.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gql.tada/cli-utils": minor
+"gql.tada": minor
+---
+
+Support a second argument in `graphql.persisted` which represents the `DocumentNode` to create the possibility of persisted-operations that still lock down the origin. Before this change we had to advise folks to use APQ.
```

Comment:
```suggestion
Support a second argument in `graphql.persisted` which accepts a `TadaDocumentNode` rather than passing a generic. This allows the document node to not be hidden, to still generate `documentId` via `gql.tada` without having to hide the document during runtime.
```